### ISSUE TITLE
2C - Parada Play: "Explain that if feet touch we stop."

### ISSUE THREAD

**Issue by thomasfischersm** [andreimoment/tangomanual#74]
I've read all the missions through 2C. They read very tight and juicy. I can clearly visualize what students would do and how they are having fun.

The parada play says "Explain that if feet touch we stop." Obviously, you know a lot more Tango than my little thing. That doesn't seem right. It seems that the upper body or energy lead the stop. For one, that's what I've always heard in classes. For the other, it seems the follower should focus and connect at the chest, not pay attention to all kinds of little other random things. And a final point: Doing a backwards ocho and touching the foot could also lead to a foot drag that enters a turn.

Maybe, this is something for me to learn more about Tango.

**Comment by Tommykurtsmith** [comment]
I agree with the general idea on how a parada is led. But since this is only a level 2 class, perhaps the tips are designed to focus on what the dancer needs at the moment to avoid delving too deep.

I wonder if subsequent levels could be added down the line for depth on a subject, like "Parada Revisited"
